dev-system/Dockerfile.ecs

```diff
@@ -1,4 +1,14 @@
 FROM eclipse-temurin:17
 
+RUN apt-get update
+RUN apt-get install zip -y
+
 COPY ./target/cmr-dev-system-0.1.0-SNAPSHOT-standalone.jar /app/cmr-standalone.jar
-WORKDIR /app
+
+RUN echo 'Unzip cmr files for drift compatibility -- so it can find the migration files'
+RUN mkdir -p /app/cmr-files
+COPY ./target/cmr-dev-system-0.1.0-SNAPSHOT-standalone.jar /app/cmr-files/cmr-copy.jar
+
+RUN cd /app/cmr-files && unzip cmr-copy.jar
```

Comment:
Do we need to unzip the entire dev-system jar or can we just selectively copy the needed files? How is this going to impact the size of our docker image?

Reply:
Not sure if you can directly copy folder dirs from the jar. I thought you had to unzip it all first to get to the folders... I could also remove/delete unused folders. Right now, we'll need boostrap's migration folder, ingest's migration folder and metadata-db's migration folder

Reply:
This wouldn't be copying files out of the jar though, since this is the Dockerfile, it just has the entire CMR directory available to it when making the image. So you could do COPY ../metadata-db-app/blah/blah/blah/migrate or whatever

Reply:
ok I have changed the build to get the migration files and this docker will now use only the smaller dir of just migration files